Makefile.toml

```diff
@@ -21,6 +23,12 @@ private = true
 condition = { env = { "TEST_SUITE" = "xpack" }, env_not_set = ["ELASTICSEARCH_URL"] }
 env = { "ELASTICSEARCH_URL" = "https://elastic:changeme@localhost:9200" }
 
+[tasks.set-elasticsearch-crate-env]
```

Comment:
This task doesn't seem to be used. Leftover from a previous iteration?

Reply:
Yes, good catch! Will remove

Reply:
removed